server/controllers/stock/reports/stock_exit_patient.receipt.handlebars

```diff
@@ -20,8 +20,10 @@
             {{translate 'STOCK.RECEIPT.EXIT_PATIENT'}}
           </span> <br>
           <strong>{{details.patient_reference}} - {{details.patient_display_name}}</strong> <br>
-          <span>{{details.document_reference}}</span> <br>
-          <small>{{date details.date}}</small> <br>
+          <div class="row">
+            <small>{{> barcode value=details.barcode}}</small>
+            <br/><br/>
```

Comment:
Could you remove this `<br/><br/>`, i think it is not necessary to have these spaces